### PR TITLE
Updated to beta.16

### PR DESCRIPTION
**Pull request by skmedix**

### CODE REVIEW

File: js/package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
-    "name": "@fof/sitemap",
-    "version": "0.0.0",
     "private": true,
+    "name": "@fof/sitemap",
     "dependencies": {
         "flarum-webpack-config": "^0.1.0-beta.10",
-        "webpack": "^4.26.0",
```
Comment (askvortsov1) [inline js/package.json:7]: Are these changes necessary for this PR?

**Review by askvortsov1 — COMMENTED** [review]
I would recommend dropping all the FoF components stuff, and using the `buildSettingComponent` method on `ExtensionPage` where necessary.

File: js/package.json (hunk 1 above)
Comment (skmedix) [reply js/package.json:7]: It's for consistency with other fof extensions

**Comment by skmedix** [comment]
@askvortsov1 I dropped `fof/components`, I didn't amend this commit for better diff. Feel free to squash this PR on merge

**Review by askvortsov1 — APPROVED** [review]

**Review by askvortsov1 — CHANGES_REQUESTED** [review]
Actually, a few more changes needed to avoid deprecated stuff:

- The `app` helper is used in the migration. TBH, that migration should be updated to use the Flarum migration helper:
```
Migration::addSettings([
    'fof-sitemap.mode' => 'run',
    'fof-sitemap.frequency' => 'daily'
]);
```

In extend.php, an old closure-based extender is used to register service providers. These will be removed in stable; the service provider extender should be used instead.

**Review by askvortsov1 — APPROVED** [review]

File: js/src/admin/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import app from 'flarum/app';
+import app from 'flarum/common/app';
```
Comment (davwheat) [inline js/src/admin/index.js:1]: This isn't shared between forum and admin, so we should use `admin` here.

```suggestion
import app from 'flarum/admin/app';
```

**Review by davwheat — CHANGES_REQUESTED** [review]

**Review by davwheat — APPROVED** [review]
Not tested locally, but all looks good to me!

I trust Sasha's review :P

**Pull request merged**